src/backend/distributed/connection/placement_connection.c

```diff
@@ -432,6 +432,31 @@ StartPlacementListConnection(uint32 flags, List *placementAccessList,
 }
 
 
+MultiConnection *
+GetPlacementListConnectionIfCached(int flags, List *placementAccessList,
```

Comment:
Missing function comment

Reply:
Maybe use `GetConnectionIfPlacementAccessedInXact()`
`GetConnectionIfAnyPlacementsAccessedInXact()` as we discussed. @marcocitus any comment?

Reply:
Sounds good, slight preference for  GetConnectionIfPlacementAccessedInXact`